plugins/woocommerce/.eslintignore

```diff
@@ -1,20 +1,9 @@
 *.min.js
 
-/assets/js/accounting/**
-/assets/js/flexslider/**
-/assets/js/jquery-blockui/**
-/assets/js/jquery-cookie/**
-/assets/js/jquery-flot/**
-/assets/js/jquery-payment/**
-/assets/js/jquery-qrcode/**
-/assets/js/jquery-serializejson/**
-/assets/js/jquery-tiptip/**
-/assets/js/jquery-ui-touch-punch/**
-/assets/js/js-cookie/**
-/assets/js/photoswipe/**
-/assets/js/prettyPhoto/**
-/assets/js/round/**
-/assets/js/select2/**
-/assets/js/selectWoo/**
-/assets/js/stupidtable/**
-/assets/js/zoom/**
\ No newline at end of file
+/.wireit/**
```

Comment:
The lines that you removed here are for jQuery libraries that were moved to `client/legacy/js`. The move hasn't caused a problem with the linting because the libraries haven't been updated. But we should exclude those libraries in their new location.

Reply:
There is `/assets/**` and `/client/**` in the exclusion list, which should include those paths. Unless you're saying we should not exclude everything under `assets` and under `client`, and list each of those libraries individually?

Reply:
> Unless you're saying we should not exclude everything under assets and under client, and list each of those libraries individually?

`client/legacy/js` contains some folders of code that we maintain. Replace `assets/` with `client/legacy/`. The `assets` folder is a build folder so it should be find to be excluded.

Reply:
Updated the ignore list with https://github.com/woocommerce/woocommerce/pull/46214/commits/fd6948b4be9b73618bf7e77b7cd34a1b6920c525. I hope I got them right this time.

Reply:
> I hope I got them right this time.

The list looks good.